akka-docs/rst/scala/code/docs/serialization/SerializationDocSpec.scala

```diff
@@ -218,7 +218,10 @@ package docs.serialization {
       //#actorref-serializer
 
       //#external-address
-      object ExternalAddress extends ExtensionKey[ExternalAddressExt]
+      object ExternalAddress extends ExtensionId[ExternalAddressExt] {
+        override def createExtension(system: ExtendedActorSystem) =
```

Comment:
It's an abstract method, which already returns the `ExternalAddressExt` from the `ExtensionId`, not sure that adds anything useful here, if we'd return the wrong thing we get compilation error. I'd agree if it was a public api, but this is a sample of how a user would write an extension.